docker/api/Dockerfile

```diff
@@ -12,27 +12,30 @@ LABEL org.opencontainers.image.vendor="Cybersecurity and Infrastructure Security
 
 ARG CISA_UID=421
 ENV CISA_HOME="/home/cisa"
-ENV ECHO_MESSAGE="Hello World from Dockerfile"
 
 RUN addgroup --system --gid ${CISA_UID} cisa \
   && adduser --system --uid ${CISA_UID} --ingroup cisa cisa
 
-RUN apk --update --no-cache add \
-ca-certificates \
-openssl \
-py-pip
+# RUN apk --update --no-cache add \
+
+RUN apt-get update && \
+ apt-get install wget
+
+# ca-certificates \
+# openssl \
+# py-pip
 
 WORKDIR ${CISA_HOME}
 
-RUN wget -O sourcecode.tgz https://github.com/cisagov/skeleton-python-library/archive/v${VERSION}.tar.gz && \
+RUN wget -O sourcecode.tgz https://github.com/cisagov/li-pca-app/archive/refs/tags/v0.0.1-rc2.tar.gz && \
```

Comment:
Why is the version hardcoded?  The `VERSION` arg still exists above.

Reply:
Good catch. We didn't intend on that staying there. I'll get it updated. Thanks!

Reply:
Addressed in d3d8a98a4f7f80611fa5ff354fa321633e624a8f